use-cases.md

```diff
@@ -53,40 +53,58 @@ In case your device is a Fiscal Machine \(no matter whether it is a virtual or a
 
 In case your device is a Key Cutter, you would get a command containing [Key cutter command data](operations/integrations.md#key-cutter-command-data).
 
-## Point of sale systems
+## Point of Sale
 
-Point of Sale systems \(POS\) need to be able to create charges in Mews. With Mews, charges are placed on a customer as rooms do not pay bills, customers do. Therefore, a POS system needs to identify/search customers where revenue items can be posted. By default, only positive charges are allowed.
+Traditionally, a Point of Sale system (POS) will send totals from the system to a room or reservation located inside the Property Management System (PMS). 
 
-### Inital Setup
+Unlike the example above, charges are not sent to rooms or reservations in Mews but the customer profile inside the PMS that are active and attached to a reservation (checked in). The items that are sent from the POS to Mews are called Orders.
 
-The integration should use the [Get all services](operations/services.md#get-all-services) operation to retrieve all services the property has configured in Mews. Once all services are retrieved, the service which you would like all charges to be sent under should be selected. Ideally this would be done via the UI in the setup page. Alternatively, a field for the `ServiceId` to be entered in can be used.
+The receipts that have been finalized in the POS can be sent to Mews to allow for end of day balancing. In Mews, these are called Outlets. 
 
-In the POS system, accounting categories usually exist such as entree, main, dessert, beverage or alcohol. In order for the revenue to be correctly reported in Mews with these categories, they should be correctly mapped against accounting categories in Mews. The POS integration should use the [Get all account categories](operations/finance.md#get-all-accounting-categories) operation to retrieve a list of all `AccountingCategories` which the property has in Mews. These should be mapped via a UI in the POS.
+By default, only positive charges are allowed. 
+
+Mews provides a guide for properties to follow before using a Point of Sale integration, which can be viewed [here]([operations/services.md#get-all-services](https://help.mewssystems.com/hc/en-us/articles/360002080037-Point-of-sale-integrations-for-Commander)).  
+
+### Initial Setup
```

Comment:
Inconsistent usage of upper case.
e.g Searching customers vs Initial Setup

Reply:
For consistency of rest of use cases, changed to `Initial setup`